pool/app/custom_field/entity.ml

```diff
@@ -687,3 +687,20 @@ module Write = struct
     }
   [@@deriving eq, show]
 end
+
+let group_fields groups fields =
+  let partition fields group =
```

Comment:
With switched input arguments, "fields" can be removed

```ocaml
let partition group = 
  let open CCFun in
  CCList.partition
    (group_id %> CCOption.map_or ~default:false (Id.equal group.Group.id))
in
```

Reply:
@mabiede ping